app/controllers/api/v0/followers_controller.rb

```diff
@@ -6,9 +6,9 @@ class FollowersController < ApiController
 
       def organizations
         @follows = Follow.
-          where(followable_id: @user.organization_id, followable_type: "Organization").
+          where(followable_id: @user.organizations.pluck(:id), followable_type: "Organization").
```

Comment:
I think I'd prefer `@user.organization_ids`, both generate the following SQL:

```
SELECT "organizations"."id" 
FROM "organizations" 
INNER JOIN "organization_memberships" 
ON "organizations"."id" = "organization_memberships"."organization_id" 
WHERE "organization_memberships"."user_id" = $1
```

Reply:
done :)